recipes-metrological/webbridge/webbridge_git.bb

```diff
@@ -87,6 +89,12 @@ EXTRA_OECMAKE += "\
 "
 # ----------------------------------------------------------------------------
 
+do_configure_append() {
+    for f in `find ${B} -name "*.make"`; do
+        sed -i -e 's#-I${STAGING_INCDIR_NATIVE}##g' $f
+    done
+}
+
```

Comment:
I think the better fix for this may be to remove these lines from CMakeLists.txt:

# include staging directory.
if (CMAKE_CROSSCOMPILING)
      foreach (path ${CMAKE_FIND_ROOT_PATH})
	      include_directories (${path}/usr/include/)
      endforeach (path)
endif(CMAKE_CROSSCOMPILING)

Paths defined via CMAKE_FIND_ROOT_PATH are intended for internal use by cmake, not to be unconditionally added to include_directories?